<?xml version="1.0" encoding="UTF-8"?>

<launch>

	<arg name="description" default="explicitaccel_microaccel"/>
	
	<arg name="hwil" default="true"/>
	<arg name="max_deaccel_lead" default="-4.5"/>
	<arg name="use_target_speed" default="true"/>
	<arg name='readonly' default='true'/>

	<param name="description" value="$(arg description)"/>
		
	<param name="/max_deaccel_lead" value="$(arg max_deaccel_lead)"/>
	<param name="/use_target_speed" value="$(arg use_target_speed)"/>
	
	<param name="/readonly" value="$(arg readonly)"/>
	<param name="/hwil" value="$(arg hwil)"/>
<!--
	<node pkg="can_to_ros" type ="subs_fs" name="subs_fs" output="screen" if="$(arg hwil)"/> 
	<node pkg="can_to_ros" type="lead_info" name="lead_info" output="screen" if="$(arg hwil)"/>
	<node pkg="can_to_ros" type="simple_mini_car_from_lead_distance" name="simple_mini_car_from_lead_distance" output="screen" if="$(arg hwil)"/>
-->
    <include file="$(find can_to_ros)/launch/toyota_interface.launch">
		<param name="description" value="$(arg description)"/>
	</include>

<!--
	<node pkg="can_to_ros" type="vehicle_interface_nissan" name="vehicle_interface_nissan" output="screen" />
	<node pkg="can_to_ros" type="subs_fs_test" name="subs_fs_test" output="screen" />
	<node pkg="can_to_ros" type ="rosbag_record.sh" name="bashscript2" output="screen" args="$(arg description) $(arg hwil)" /> 
-->

	<node pkg="explicitaccel_microaccel" type="explicitaccel_microaccel" name="explicitaccel_microaccel_node" output="screen">
		<remap from="/lead_dist" to="/lead_dist_869"/>
	</node>


</launch>
